daemon/cmd/policy.go

```diff
@@ -332,7 +332,9 @@ func (d *Daemon) policyAdd(sourceRules policyAPI.Rules, opts *policy.AddOptions,
 	// revision.
 	endpointsToBumpRevision := policy.NewEndpointSet(allEndpoints)
 
-	endpointsToRegen := policy.NewEndpointSet(nil)
+	addEndpointsToRegen := policy.NewEndpointSet(nil)
+	deletedEndpointsToRegen := policy.NewEndpointSet(nil)
```

Comment:
As a single endpoint may select multiple rules I would not collate the two here.